src/app/modals/search-modal/search-modal.component.pug

```diff
@@ -73,8 +77,7 @@
                           label.custom-control-label((click)="toggleFilter(filter)")
                             | {{ filter.name }} {{ filter.count ? '(' + filter.count +')' : '' }}
                         ul.list-group.list-group-child(*ngIf="filter.children")
-                          li.list-group-item(*ngFor="let child of filter.children")
-                            .custom-control.custom-checkbox
+                            .custom-control.custom-checkboxli.list-group-item(*ngFor="let child of filter.children")
```

Comment:
should this be `.custom-control.custom-checkbox.list-group-item`?

Reply:
Good call Gayla, accidentally moved it clicked around the file. Thank you!